nucleus/distributions/pom.xml

```diff
@@ -92,69 +99,66 @@
-                            <phase>process-resources</phase>
+                            <id>create-artifact</id>
+                            <phase>prepare-package</phase>
+                            <goals>
+                                <goal>single</goal>
+                            </goals>
                             <configuration>
-                                <executable>${stage.dir}/${nucleus.install.dir.name}/bin/nadmin</executable>
-                                <commandlineArgs>${create-domain.args}</commandlineArgs>
+                                <descriptors>
+                                    <descriptor>src/main/assembly/${project.artifactId}.xml</descriptor>
+                                </descriptors>
+                                <ignoreMissingDescriptor>false</ignoreMissingDescriptor>
+                                <finalName>${stage.dir.name}</finalName>
+                                <attach>false</attach>
+                                <appendAssemblyId>false</appendAssemblyId>
                             </configuration>
                         </execution>
-
+                    </executions>
+                </plugin>
+                <plugin>
+                    <groupId>org.codehaus.mojo</groupId>
+                    <artifactId>exec-maven-plugin</artifactId>
+                    <executions>
                         <execution>
-                            <id>default-zip</id>
-                            <phase>package</phase>
+                            <id>create-domain</id>
+                            <phase>prepare-package</phase>
+                            <goals>
+                                <goal>exec</goal>
+                            </goals>
                             <configuration>
-                                <excludes>**/.gitkeep_empty_dir</excludes>
-                                <duplicate>preserve</duplicate>
-                                <filesets>
-                                    <fileset>
-                                        <dir>${stage.dir}</dir>
-                                        <fileMode>755</fileMode>
-                                        <includes>**/bin/*</includes>
-                                        <includes>**/bin/*.bat</includes>
-                                        <includes>**/lib/nadmin*</includes>
-                                    </fileset>
-                                    <fileset>
-                                        <dir>${stage.dir}</dir>
-                                        <excludes>**/bin/*</excludes>
-                                        <excludes>**/bin/*.bat</excludes>
-                                        <excludes>**/lib/nadmin*</excludes>
-                                    </fileset>
-                                </filesets>
+                                <executable>${stage.dir}/${nucleus.install.dir.name}/bin/nadmin</executable>
```

Comment:
Failed on Windows

Reply:
Here we should use script without extension on UNIX, and `.bat` on Windows. 

Hint: I make a quick build with adding extension via profile - all builds. May be found a better way ;)

Reply:
It should be fixed now

Reply:
Yes, now it works.